nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractSyslogProcessor.java

```diff
@@ -29,34 +29,28 @@
     public static final AllowableValue TCP_VALUE = new AllowableValue("TCP", "TCP");
     public static final AllowableValue UDP_VALUE = new AllowableValue("UDP", "UDP");
 
-    public static final PropertyDescriptor PROTOCOL = new PropertyDescriptor
+    public static final PropertyDescriptor PROTOCOL_PROP = new PropertyDescriptor
             .Builder().name("Protocol")
             .description("The protocol for Syslog communication.")
             .required(true)
             .allowableValues(TCP_VALUE, UDP_VALUE)
```

Comment:
Since protocol uses allowable values, there isn't a way for a user to enter expression language

Reply:
Yup, you're right will adjust